### PR TITLE
Feature/35070

### PR DESCRIPTION
**Pull request by szafran89**
Github issue #69

### CODE REVIEW

File: components/03-modules/header/header.hbs [hunk 1]
```diff
@@ -1,17 +1,95 @@
...
+<header class="header">
+    {{ render '@marketing-bar' }}
+    <div class="header__top-bar">
+        <div class="container header__top-bar-wrapper">
+            <div class="header__contact-info">
+                + 48 68 433 00 00
+                <a class="header__contact-info-link" href="#">Contact</a>
+            </div>
+            <div class="header__links">
+                <a class="header__links-item" href="#">
```
Comment (kruchy8) [inline components/03-modules/header/header.hbs]: Try extracting data to config.js and using loop for this `header__link-item`

File: components/03-modules/header/header.hbs [hunk 2]
```diff
@@ -1,17 +1,95 @@
...
+                    Create account
+                </a>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="header__wrapper">
+            {{ render '@logo' logo }}
+            <div class="header__icons">
+                <a class="header__icons-item" href="#">
```
Comment (kruchy8) [inline components/03-modules/header/header.hbs]: Same here, try using loop.

File: components/03-modules/header/_featured-info.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    &__icon {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        margin-right: 8px;
+
+        &-svg {
+            width: 24px;
+            height: 24px;
+            fill: $color-primary;
```
Comment (kruchy8) [inline components/03-modules/header/_featured-info.scss]: move color-primary to internal variable and then use it here

File: components/03-modules/header/_header.scss [hunk 4]
```diff
@@ -1,32 +1,231 @@
...
+
+            &:focus,
+            &:hover {
+                text-decoration: $header__text-decoration;
+            }
+        }
+    }
+
+    &__links {
+        margin: 8px 0;
```
Comment (kruchy8) [inline components/03-modules/header/_header.scss]: Internal variable

File: components/03-modules/header/_header.scss [hunk 5]
```diff
@@ -1,32 +1,231 @@
...
+        font-size: $header__font-size;
+
+        &-item {
+            padding: 0 16px;
+            border-left: 1px solid $header__text-border-color;
+            line-height: 14px;
```
Comment (kruchy8) [inline components/03-modules/header/_header.scss]: Use internal variables for this `px`values

File: components/03-modules/header/header.hbs [hunk 6]
```diff
@@ -1,17 +1,95 @@
-<header class="header container">
-    {{ render '@logo' }}
+<header class="header">
+    {{ render '@marketing-bar' }}
+    <div class="header__top-bar">
+        <div class="container header__top-bar-wrapper">
+            <div class="header__contact-info">
+                + 48 68 433 00 00
```
Comment (kruchy8) [inline components/03-modules/header/header.hbs]: I believe this kind of info should come from config.js

File: components/02-elements/marketing-bar/_marketing-bar.scss [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+$marketing-bar__button-spacing   : 12px !default;
+$marketing-bar__button-icon-color: $white !default;
+
+.marketing-bar {
+    position: relative;
+    display: flex;
+    height: 48px;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 8px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/marketing-bar/_marketing-bar.scss]: fixed.

File: components/02-elements/minicart-trigger/_minicart-trigger.scss [hunk 8]
```diff
@@ -1,17 +1,21 @@
...
 $minicart-trigger__counter-color     : $white !default;
+$minicart-trigger__counter-size      : 20px !default;
 
 .minicart-trigger {
-    background: $minicart-trigger__background;
     position: relative;
-    margin: 0;
+    display: block;
+    padding: 12px;
+    margin: 0 8px 0 0;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/minicart-trigger/_minicart-trigger.scss]: fixed

File: components/02-elements/minicart-trigger/_minicart-trigger.scss [hunk 9]
```diff
@@ -20,22 +24,23 @@ $minicart-trigger__counter-color     : $white !default;
 
     &__icon {
         display: block;
-        width: $minicart-trigger__icon-spacing;
-        height: $minicart-trigger__icon-spacing;
+        width: $minicart-trigger__icon-size;
+        height: $minicart-trigger__icon-size;
+        padding: 2px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/minicart-trigger/_minicart-trigger.scss]: added

File: components/02-elements/side-menu-trigger/_side-menu-trigger.scss [hunk 10]
```diff
@@ -1,19 +1,18 @@
...
-$side-menu-trigger__text-color        : $gray !default;
 $side-menu-trigger__line-background   : $gray-dark !default;
-$side-menu-trigger__line-height       : 3px !default;
-$side-menu-trigger__line-border-radius: 3px !default;
+$side-menu-trigger__line-height       : 2px !default;
+$side-menu-trigger__line-border-radius: 2px !default;
 
 .side-menu-trigger {
-    padding: 0;
+    padding: 8px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/side-menu-trigger/_side-menu-trigger.scss]: added

File: components/02-elements/side-menu-trigger/_side-menu-trigger.scss [hunk 11]
```diff
@@ -48,22 +47,25 @@ $side-menu-trigger__line-border-radius: 3px !default;
         bottom: 0;
         left: 0;
         width: 100%;
-        color: $side-menu-trigger__text-color;
         font-size: $side-menu-trigger__font-size;
+        font-weight: bold;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/side-menu-trigger/_side-menu-trigger.scss]: fixed

File: components/02-elements/side-menu-trigger/_side-menu-trigger.scss [hunk 12]
```diff
@@ -48,22 +47,25 @@ $side-menu-trigger__line-border-radius: 3px !default;
...
     }
 
     &__line {
         position: absolute;
         left: 0;
-        width: 100%;
+        right: 0;
+        width: 55%;
         height: $side-menu-trigger__line-height;
+        margin: auto;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/side-menu-trigger/_side-menu-trigger.scss]: I think it should be static not moved to variable, this element have to be margin auto all time.

File: components/03-modules/header/_featured-info.scss [hunk 13]
```diff
@@ -0,0 +1,49 @@
+$featured-info__title-fontsize   : 12px !default;
+$featured-info__subtitle-fontsize: 10px !default;
+
+.featured-info {
+    display: flex;
+    margin: 8px 16px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/03-modules/header/_featured-info.scss]: done

File: components/03-modules/header/_featured-info.scss [hunk 14]
```diff
@@ -0,0 +1,49 @@
...
+    color: $font-color-base;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/03-modules/header/_featured-info.scss]: done.

File: components/03-modules/header/_featured-info.scss (hunk 3 above)
Comment (szafran89) [reply components/03-modules/header/_featured-info.scss]: done

File: components/03-modules/header/_header.scss [hunk 15]
```diff
@@ -1,32 +1,231 @@
-$header-padding   : 15px !default;
-$header-margin    : 15px !default;
-$header-background: $white !default;
+$header__text-border-color: $gray-dark !default;
+$header__border-color     : $gray !default;
+$header__font-size        : 12px !default;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/03-modules/header/_header.scss]: sure :)

File: components/03-modules/header/_header.scss [hunk 16]
```diff
@@ -1,32 +1,231 @@
...
+    &__contact-info {
+        margin: 8px 0;
+        font-size: $header__font-size;
+
+        &-link {
+            display: inline-flex;
+            margin-left: 8px;
+            padding-left: 8px;
+            border-left: 1px solid $header__text-border-color;
+            line-height: 14px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/03-modules/header/_header.scss]: done.

File: components/03-modules/header/_header.scss (hunk 5 above)
Comment (szafran89) [reply components/03-modules/header/_header.scss]: fixed

File: components/03-modules/header/_header.scss (hunk 4 above)
Comment (szafran89) [reply components/03-modules/header/_header.scss]: fixed

File: components/03-modules/header/_header.scss [hunk 17]
```diff
@@ -1,32 +1,231 @@
...
+        padding: 2px;
+        fill: $gray-dark;
+        cursor: pointer;
+    }
+
+    &__menu-trigger {
+        display: flex;
+        flex: 1 20%;
+        order: 4;
+        height: 49px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/03-modules/header/_header.scss]: no, there is border-top additionally. I can add another wrapper div but for now let's keep this shape.

File: components/03-modules/header/_header.scss [hunk 18]
```diff
@@ -1,32 +1,231 @@
...
-        > *:last-child {
-            margin-right: 0;
+    &__search-wrapper {
+        display: flex;
+        flex-wrap: nowrap;
+        flex-direction: column;
+        flex: 1 70%;
+        order: 5;
+        justify-content: center;
+        width: 100%;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/03-modules/header/_header.scss]: yes, I find it's ok.

File: components/03-modules/header/header.hbs (hunk 6 above)
Comment (szafran89) [reply components/03-modules/header/header.hbs]: dane.

File: components/03-modules/header/header.hbs (hunk 2 above)
Comment (szafran89) [reply components/03-modules/header/header.hbs]: done.

File: components/03-modules/header/header.hbs (hunk 1 above)
Comment (szafran89) [reply components/03-modules/header/header.hbs]: done.

File: components/03-modules/header/_header.scss (hunk 17 above)
[the remark this answers is not in the document]
Comment (kruchy8) [reply components/03-modules/header/_header.scss]: ok

File: components/03-modules/header/_header.scss (hunk 18 above)
[the remark this answers is not in the document]
Comment (kruchy8) [reply components/03-modules/header/_header.scss]: ok

File: components/02-elements/side-menu-trigger/_side-menu-trigger.scss (hunk 12 above)
[the remark this answers is not in the document]
Comment (kruchy8) [reply components/02-elements/side-menu-trigger/_side-menu-trigger.scss]: ok, if you say so :P

File: components/02-elements/minicart-trigger/_minicart-trigger.scss [hunk 19]
```diff
@@ -1,17 +1,21 @@
...
+$minicart-trigger__counter-background: $color-primary !default;
 $minicart-trigger__counter-color     : $white !default;
+$minicart-trigger__counter-size      : 20px !default;
 
 .minicart-trigger {
-    background: $minicart-trigger__background;
     position: relative;
-    margin: 0;
+    display: block;
+    padding: 12px;
```
[the remark this answers is not in the document]
Comment (kruchy8) [reply components/02-elements/minicart-trigger/_minicart-trigger.scss]: ?

File: components/02-elements/marketing-bar/_marketing-bar.scss [hunk 20]
```diff
@@ -0,0 +1,41 @@
+$marketing-bar__background-color : $color-primary !default;
+$marketing-bar__font-size        : $font-size-small !default;
+$marketing-bar__button-spacing   : 12px !default;
+$marketing-bar__button-icon-color: $white !default;
+
+.marketing-bar {
+    position: relative;
+    display: flex;
+    height: 48px;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/marketing-bar/_marketing-bar.scss]: fixed

File: components/02-elements/marketing-bar/_marketing-bar.scss [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+    flex-direction: column;
+    justify-content: center;
+    padding: 0 8px;
+    font-size: $marketing-bar__font-size;
+    font-weight: bold;
+    background-color: $marketing-bar__background-color;
+    color: $white;
```
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/marketing-bar/_marketing-bar.scss]: done.

File: components/02-elements/minicart-trigger/_minicart-trigger.scss (hunk 19 above)
[the remark this answers is not in the document]
Comment (szafran89) [reply components/02-elements/minicart-trigger/_minicart-trigger.scss]: done

**Pull request merged**